src/main/java/org/jabref/gui/EntryTypeView.java

```diff
@@ -79,7 +79,10 @@ public EntryTypeView(LibraryTab libraryTab, DialogService dialogService, Prefere
                   .load()
                   .setAsDialogPane(this);
 
-        ControlHelper.setAction(generateButton, this.getDialogPane(), event -> viewModel.runFetcherWorker());
+        ControlHelper.setAction(generateButton, this.getDialogPane(), event -> {
+            viewModel.runFetcherWorker();
+            visualizer.initVisualization(viewModel.idFieldValidationStatus(), idTextField, true);
+        });
```

Comment:
Sorry, but this does not seem to work, since the generate button is disabled by default and the validation does only check if the button isNotBlank.

Reply:
Fixed,